bitnami/minio/templates/networkpolicy.yaml

```diff
@@ -74,10 +74,7 @@ spec:
         {{- end }}
         {{- $extraIngress := coalesce .Values.networkPolicy.extraIngress .Values.networkPolicy.extraFromClauses }}
         {{- if $extraIngress }}
-        {{- toYaml $extraIngress | nindent 8 }}
+        {{- toYaml $extraIngress | nindent 6 }}
         {{- end }}
       {{- end }}
-    {{- if .Values.networkPolicy.extraIngress }}
-    {{- include "common.tplvalues.render" ( dict "value" .Values.networkPolicy.extraIngress "context" $ ) | nindent 4 }}
-    {{- end }}
```

Comment:
Using the template chart as reference, which is used to model all of the charts in our catalog, the code block we should keep is the one that has been removed:
https://github.com/bitnami/charts/blob/60473dae2a7bfcd6f702482f488f080913e36172/template/CHART_NAME/templates/networkpolicy.yaml#L69-L71

The value `networkPolicy.extraFromClauses` is a legacy value that was introduced before we defined the standard values for our networkPolicy templates.

I would suggest to modify the ingress block to look like this:
```yaml
        {{- if .Values.networkPolicy.extraFromClauses }}
        {{- toYaml .Values.networkPolicy.extraFromClauses | nindent 8 }}
        {{- end }}
      {{- end }}
    {{- if .Values.networkPolicy.extraIngress }}
    {{- include "common.tplvalues.render" ( dict "value" .Values.networkPolicy.extraIngress "context" $ ) | nindent 4 }}
    {{- end }}
```